test/lorawan_tests.erl

```diff
@@ -0,0 +1,158 @@
+-module(lorawan_tests).
+
+%% ------------------------------------------------------------------
+%% EUNIT Tests
+%% ------------------------------------------------------------------
+-ifdef(TEST).
+-include_lib("eunit/include/eunit.hrl").
+
+-define(RETIRED_NETID, 16#200010).
+
+id_test() ->
```

Comment:
`id_test()` seems identical to the eunit other than using fully qualified function names here.

If there's a reason for both, please add a comment with the justification.

Reply:
I waffled on this one.  Originally I intended to write all unit tests in an external file.  However AFAIK Erlang provides no mechanism to keep internal functions private, so I switched back to unit tests within the file.